augur/measurements/export.py

```diff
@@ -207,8 +207,13 @@ def run(args):
         print("ERROR: Cannot create measurements JSON without valid groupings", file=sys.stderr)
         sys.exit(1)
 
+    # Convert deprecated single threshold value to list if "thresholds" not provided
+    single_threshold = collection_config.pop("threshold", None)
+    if single_threshold is not None and "thresholds" not in collection_config:
+        collection_config["thresholds"] = [single_threshold]
```

Comment:
I'm not clear on the expected deprecation/support story here with regards to Auspice. There's no existing Auspice version that uses `thresholds` but with this Augur will only produce `thresholds`, thus making them talk past each other? It seems like this arrangement is likely to lead to harsh compatibility constraints, even once there is a release of Auspice that uses `thresholds`.

It seems to me that Augur could instead produce `threshold` as a single number when only one threshold is provided, thus only raising compatibility issues if multiple thresholds are actually provided. Then any existing usage with a single threshold (i.e. all uses of it currently, right?) won't have any compatibility issues with Augur and Auspice.

Reply:
Yeah, @huddlej and I had a brief conversation about this in [Slack](https://bedfordlab.slack.com/archives/C2G8C5LUD/p1675372891080179?thread_ts=1675294755.511109&cid=C2G8C5LUD). 

My thought here is that we will want to drop the deprecated `threshold` key as some point, so might as well do it now when there aren't that many (known) users of the command/feature.

Reply:
[continuing this discussion in another thread](https://github.com/nextstrain/augur/pull/1148#discussion_r1099114446) (probably shouldn't have split the two ideas in the first place)